Individual Scripts/lower-ram-usage.reg

```diff
@@ -0,0 +1,173 @@
+Windows Registry Editor Version 5.00
+
+; lowers ram usage and process count by a lot
+[HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control]
+"SvcHostSplitThresholdInKB"=dword:04000000
```

Comment:
Setting this to such a high number might unwanted service shutdowns or instability because more services get grouped together. This should be used only if you know what you are doing.

See also:
https://www.tenforums.com/tutorials/94628-change-split-threshold-svchost-exe-windows-10-a.html